### DISCUSSION TITLE
How to run demo editor?

### DISCUSSION THREAD

**Discussion by cawa-93** [WordPress/gutenberg#49358] [Q&A]
I just tried to run `@wordpress/block-editor` in [stackblitz](https://stackblitz.com/edit/vitejs-vite-vmreeq?file=src/main.jsx) and it's not working properly. I see error in console `Uncaught TypeError: can't access property "current", shortcuts is undefined`. But I don't know what's wrong. I simply copied the demo code from [README](https://www.npmjs.com/package/@wordpress/block-editor) and expected at least something on the page.

**Comment by devfle** [comment]
Hi, without further information about your env and all your steps you've done, it is difficult to help. If you want to init an own instance of the block editor, I highly recommend checking out this awesome repository: https://github.com/getdave/standalone-block-editor
It is also well documented and takes the same approach as in the mentioned readme.

**Comment by t-hamano** [comment]
Hi @cawa-93,

```
Uncaught TypeError: can't access property "current", shortcuts is undefined
```

As for this error, it appears to be the same issue reported in #37152.

If you are interested in creating a stand-alone block editor, the source code for the following products may be helpful.

- [Blocks Everywhere](https://github.com/Automattic/blocks-everywhere): Enable the block editor in comments, bbPress, etc.
- [Isolated Block Editor](https://github.com/Automattic/isolated-block-editor): The block editor package that does not require WordPress; it is also used internally in Blocks Everywhere.
